torch/csrc/byte_order.cpp

```diff
@@ -162,6 +162,11 @@ void THP_decodeDoubleBuffer(double* dst, const uint8_t* src, THPByteOrder order,
   }
 }
 
+void THP_decodeBoolBuffer(bool* dst, const uint8_t* src, THPByteOrder order, size_t len)
+{
+  dst = (bool*)src;
```

Comment:
this looks different from other similar functions in this file

Reply:
Fixed and added a test